### PR TITLE
simple elk example

### PR DESCRIPTION
**Pull request by andreaturli**

### CODE REVIEW

File: elk.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  install.latch: $brooklyn:component("es").attributeWhenReady("service.isUp")
+  brooklyn.config:
+    kibana.elasticsearch.ip: $brooklyn:entity("es").attributeWhenReady("host.address")
+
+- type: org.apache.brooklyn.entity.webapp.tomcat.Tomcat8Server
+  id: jboss
+  brooklyn.config:
+    children.startable.mode: background_late
+  brooklyn.children:
+  - type: logstash-child
```
Comment (aledsage) [inline elk.yaml:22]: where is `logstash-child` defined?

File: elk.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: elk
+description: |
+       Simple ELK stack deployment: it installs ES, Kibana and lostash as a child of Apache Tomcat 8
+services:
+- type: org.apache.brooklyn.entity.nosql.elasticsearch.ElasticSearchNode
+  id: es
+  name: elasticsearch
+  brooklyn.config:
+    install.version: 1.4.4
+- type: kibana-standalone
```
Comment (aledsage) [inline elk.yaml:10]: Where is kibana-standalone defined?

File: elk.yaml (hunk 1 above)
Comment (andreaturli) [inline elk.yaml:22]: you need to add this [catalog.bom](http://catalog.brooklyn.io/#blueprint/andreaturli/brooklyn-logstash)

File: elk.yaml (hunk 2 above)
Comment (andreaturli) [inline elk.yaml:10]: you need to add this [catalog.bom](http://catalog.brooklyn.io/#blueprint/andreaturli/brooklyn-kibana)

**Comment by andreaturli** [comment]
@aledsage notice you will need [logstash.bom](http://catalog.brooklyn.io/#/blueprints/andreaturli/brooklyn-logstash), [kibana](http://catalog.cloudsoft.io/#/blueprints/andreaturli/brooklyn-kibana) and possibly [elasticsearch.bom](http://catalog.brooklyn.io/#/blueprints/Graeme-Miller/brooklyn-elasticsearch)

File: elk.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- type: org.apache.brooklyn.entity.webapp.tomcat.Tomcat8Server
+  id: tomcat8
+  brooklyn.config:
+    children.startable.mode: background_late
+  brooklyn.children:
+  - type: logstash-child
+    name: logstash-child
+    brooklyn.config:
+      logstash.elasticsearch.ip: $brooklyn:entity("es").attributeWhenReady("host.address")
```
Comment (aledsage) [inline elk.yaml:25]: This should be `logstash.elasticsearch.host`, I think.

**Comment by aledsage** [comment]
Hi @andreaturli - unfortunately this example fails for me when deployed to AWS (with the `logstash.elasticsearch.host` config key name fixed).

---

TL;DR: the host IP needs to be in double quotes in the config file.

My config file (`/etc/logstash/conf.d/01-basic.conf`) says:
    input { file { path => "/home/users/aled/brooklyn-managed-processes/apps/vQFu0YPI/entities/Tomcat8Server_Fzw0b9NL/logs/catalina.out" start_position => "beginning" } }  output { elasticsearch { host => 54.170.241.179 protocol => "http" embedded => false } }

The log file (`/var/log/logstash/logstash.log`) says:
    {:timestamp=>"2016-02-17T22:55:07.156000+0000", :message=>"Error: Expected one of #, } at line 1, column 208 (byte 208) after output { elasticsearch { host => 54.170"}
    {:timestamp=>"2016-02-17T22:55:07.172000+0000", :message=>"You may be interested in the '--configtest' flag which you can\nuse to validate logstash's configuration before you choose\nto restart a running system."}

The logstash process is not running. When I try running the commands below, it fails after a couple of minutes with the same error:

```
CONFIG_DIR=/etc/logstash/conf.d/
VERSION=1.5.4
nohup /opt/logstash/logstash-${VERSION}/bin/logstash agent -f ${CONFIG_DIR} -l /var/log/logstash/logstash.log &
```

When I put the ip address in double quotes and tried starting it again, then it worked.
